### DISCUSSION TITLE
load errror while starting plugins (report_error)

### DISCUSSION THREAD

**Discussion by mihugo** [Dan-in-CA/SIP#260] [General]
Finally getting around to do my once in a while update of my SIP install.

I'm getting the following warning when starting and the plugin manager does not work:

> Ignoring exception while loading the plugin_manager plug-in.
> cannot import name report_error

Looking in   ./plugins/plugin_manager.py
  line 23 there is the line 
from helpers import report_error

but there is no "report_error" in the  ./helpers.py file and hence the error.
Removing that line makes the plugin manager  work though that may not solve the error as there are 3 calls to the report_errors in the plugin manager.

Browsing through the repo I see that the plugin-branch that version of plugin_manager.py does not have that line though that version is also older than the one on the main branch.  I'm assuming that the plugin-branch is no longer being used.

**Comment by Dan-in-CA** [comment]
Thanks for reporting that.
I thought that I got rid of all the report error references.

The plugin manager has been updated to remove those references.

You are right the plugin_data branch is obsolete.
Dan

**Comment by mihugo** [comment]
wow! thank for your instant response.  Verified the fix works.
